content/html/concepts/images/images.md

```diff
@@ -31,7 +32,7 @@ Below is an example of how an image is commonly inserted into a page using the H
 <img src="logo.png" alt="Codecademy logo" />
 ```
 
-It is recommended to describe image dimensions using percents of the current screen size to best support both smaller mobile devices as well larger workstations. The below image would take up the top half of any screen, regardless of its size:
+It is recommended to describe image [width and height](https://www.codecademy.com/resources/docs/uiux/width-height) using percents of the current screen size to best support both smaller mobile devices as well as larger workstations. The image below would take up the top half of any screen, regardless of its size:
 
 ```html
 <img src="logo.png" alt="Codecademy logo" height="50%" width="100%" />
```

Comment:
Only absolute units (possibly only px) can be used with the `hight` and `width` attributes.

Reply:
True. I edited in 89f4e0859778eac3e9eaae8d7ccb67e1c538fd08 by embedding in the global `style` attribute.